ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/manage.yml

```diff
@@ -5,6 +5,28 @@ mas_app_ws_spec:
     jdbc: "{{ mas_appws_jdbc_binding | default( 'system' , true) }}"
   components: "{{ mas_appws_components | default({'base':{'version':'latest'}}, true) }}"
   settings:
+    deployment:
```

Comment:
Bit concerned about this?  Shouldn't this new block be templated behind a {% if condition %}, otherwise this becomes the default for everyone, not just those who are doing this new stuff?

Reply:
Are all the defaults if I do nothing differently to before going to result in a compatible Manage install to what I had before this change?